src/System.Reflection.Emit.Lightweight/src/System.Reflection.Emit.Lightweight.csproj

```diff
@@ -3,20 +3,25 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyName>System.Reflection.Emit.Lightweight</AssemblyName>
-    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <GenFacadesIgnoreMissingTypes Condition="'$(TargetGroup)'=='uapaot'">true</GenFacadesIgnoreMissingTypes>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' != 'netstandard'">true</IsPartialFacadeAssembly>
+    <GenFacadesIgnoreMissingTypes Condition="'$(TargetGroup)'=='uapaot' And '$(IsPartialFacadeAssembly)' != 'true'">true</GenFacadesIgnoreMissingTypes>
```

Comment:
Do we need to keep the `$(TargetGroup)'=='uapaot'` condition here and in other files? Should I have deleted this in #30472?